packages/base-styles/_mixins.scss

```diff
@@ -437,3 +437,53 @@
 		max-width: 100%;
 	}
 }
+
+@mixin admin-scheme($color-primary, $color-secondary, $color-toggle) {
+	--wp-admin-theme-color: #{$color-primary};
+	--wp-admin-theme-color-secondary: #{$color-secondary};
+	--wp-admin-theme-color-toggle: #{$color-toggle};
+
+	// Ideally, we should limit these computed variables,
+	// because we can"t compute colors based on CSS variables.
+	// This requires refactoring some existing styles.
+	--wp-admin-theme-color-darker-20: darken(#{$color-primary}, 20%);
+	--wp-admin-theme-color-darker-10: darken(#{$color-primary}, 10%);
+	--wp-admin-theme-color-lighter-10: lighten(#{$color-primary}, 10%);
+	--wp-admin-theme-color-lighter-40: lighten(#{$color-primary}, 40%);
+}
+
+@mixin wordpress-admin-schemes() {
+	body.admin-color-light {
+		@include admin-scheme("#0085ba", "#c75726", "#11a0d2");
```

Comment:
These values should be unquoted so they're correctly parsed as colors

Reply:
Oh thanks, I was wondering for a moment why my mixin wasn't working and I ended up with interpolation but missed the quoting (which is obviously the right fix)

Reply:
@ryelle Seems like unquoting and removing the interpolation triggers an error when running `npm run build`. Not sure I understand why sass is not converting the variables properly. Any idea?

Reply:
My comment below should help— sass doesn't process values of custom properties, so you still need to use interpolation, but around the whole value: `#{darken($color-primary, 20%)};`

Reply:
Ok, I'm trying that :)

Reply:
Seems to work, I had to keep the interpolation around the regular variables too.